broker.go

```diff
@@ -81,35 +79,36 @@ func (b *Broker) continuousQueryLoop(done chan struct{}) {
 }
 
 func (b *Broker) runContinuousQueries() {
-	next := 0
-	for {
-		// if the current node hasn't been set it's our first time or we're reset. move to the next one
-		if b.currentCQProcessingNode == nil {
-			dataNodes := b.Broker.Replicas()
-			if len(dataNodes) == 0 {
-				return // don't have any nodes to try, give it up
-			}
-			next = next % len(dataNodes)
-			b.currentCQProcessingNode = dataNodes[next]
-			next++
-		}
+	topic := b.Broker.Topic(BroadcastTopicID)
+	if topic == nil {
+		log.Println("broker cq: no topics currently available.")
```

Comment:
This can happen if we outrun the first time the topics have been talked to.  We saw this race condition in other spots in our code, so while it is strange, it can occur.  I'll fix the log message.